src/views/v2/widgets/events-list.php

```diff
@@ -92,10 +95,20 @@
 
 				<div class="tribe-events-widget-events-list__events">
 					<?php foreach ( $events as $event ) : ?>
-						<?php $this->template( 'widgets/events-list/event', [ 'event' => $event, 'display' => $display ] ); ?>
+						<?php
+						$this->template(
+							'widgets/events-list/event',
+							[
+								'event' => $event,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 3 space(s) between "'event'" and double arrow, but found 1.

<sub>hash: 023a842d578e93e9bd33a3d8a0d393a7</sub>